src/config/wallets.ts

```diff
@@ -292,6 +306,16 @@ export const supportEvmWalletObj = {
     isSupportMobileApp: false,
     ethExtension: 'enkrypt.providers.ethereum',
   },
+  [SupportWallet.DcentEvm]: {
+    img: require('/src/assets/img/icon_dcent.png'),
+    name: "D'CENT Wallet (EVM)",
+    source: SupportWallet.DcentEvm,
+    walletUrl: 'https://store.dcentwallet.com/pages/dcent-wallet-app',
+    guideUrl: 'https://userguide.dcentwallet.com',
+    isSupportBrowserExtension: false,
+    isSupportMobileApp: true,
+    ethExtension: 'ethereum',
```

Comment:
You do not have your own window wallet extension name?

Reply:
@gluneau 
The app uses ethereum.isDcentWallet to distinguish.
And due to the characteristics of our wallet, dApp is supported within the app, so I don't think there is a need to worry about conflicts with other wallets.